www/docs/src/content/docs/cli/registries.mdx

```diff
@@ -166,6 +166,21 @@ Like the `npm:` alias, the `jsr:` alias can be supressed by setting
 the `registry` config option to a different value, such as a caching
 proxy in your organization's network.
 
+#### `gh:`
+
+The `gh:` alias is a special alias that allows you to install packages
+from [GitHub Package Registry](https://github.com/features/packages).
```

Comment:
That product page url does not seem to exist and is redirecting to a GitHub Actions product page. Let's point to the docs instead:

```suggestion
from [GitHub Package Registry](https://docs.github.com/en/packages).
```